docs/index.yml

```
@@ -234,7 +234,7 @@ additionalContent:
             text: Game development with Visual Studio
           - url: https://docs.cryengine.com/display/CEPROG/C%23+Programming
             text: Learn how to use CRYENGINE to build games with C#
-          - url: http://www.monogame.net/documentation/?page=main
+          - url: https://www.monogame.net/documentation/?page=main
```

That link forward to this one:

```suggestion
          - url: https://docs.monogame.net/?page=main
```

---

Is the `page=main` parameter query necessary?
I don't see a difference when I open the page with or without it.